airbyte-tests/src/acceptanceTests/java/io/airbyte/test/acceptance/CdcAcceptanceTests.java

```diff
@@ -594,22 +591,24 @@ private WebBackendConnectionUpdate getUpdateInput(final ConnectionRead connectio
   }
 
   // can be helpful for debugging
-  private void printDbs() throws SQLException {
-    final Database sourceDb = testHarness.getSourceDatabase();
-    Set<SchemaTableNamePair> pairs = testHarness.listAllTables(sourceDb);
-    LOGGER.info("Printing source tables");
-    for (final SchemaTableNamePair pair : pairs) {
-      final Result<Record> result = sourceDb.query(context -> context.fetch(String.format("SELECT * FROM %s.%s", pair.schemaName, pair.tableName)));
-      LOGGER.info("{}.{} contents:\n{}", pair.schemaName, pair.tableName, result);
-    }
-
-    final Database destDb = testHarness.getDestinationDatabase();
-    pairs = testHarness.listAllTables(destDb);
-    LOGGER.info("Printing destination tables");
-    for (final SchemaTableNamePair pair : pairs) {
-      final Result<Record> result = destDb.query(context -> context.fetch(String.format("SELECT * FROM %s.%s", pair.schemaName, pair.tableName)));
-      LOGGER.info("{}.{} contents:\n{}", pair.schemaName, pair.tableName, result);
-    }
-  }
+  // private void printDbs() throws SQLException {
```

Comment:
The formatting is a little weird for commented code, but is easy to fix for anyone who wants to uncomment